joomla/joomla-cms
# fix IE 11 error in html5.js

Pull Request for Issue reported in CMS Google Groups 

> Seems a recent update to IE 11 has caused a JS error to pop up in validate.js.
Forms will not save if the form contains an email address.
Object doesn't support property or method 'toASCII'
`t=punycode.toASCII(t)`
Works fine in edge, chrome and Firefox, anyone else noticing this?
Steve


### Summary of Changes
Make sure that we don't try to get an attribute of null object

### Testing Instructions
USE IE 11
Apply patch 
Create a new contact in th back end and make sure that you fill the email address
Make sure that the form is saved without any errors

### Documentation Changes Required
None


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/joomla/joomla-cms/13685)
